standards/core/relay-sharding.md

```diff
@@ -74,6 +83,9 @@ The global shard with index 0 and the "all app protocols" range are treated in t
 but choosing shards in the global cluster has a higher probability of sharing the shard with other apps.
 This offers k-anonymity and better connectivity, but comes at a higher bandwidth cost.
 
+Since the introduction of [the Waku Network](https://github.com/vacp2p/rfc-index/blob/main/waku/standards/core/64/network.md),
+it is RECOMMENDED that apps choose a shard within the defined range for that network.
```

Comment:
Hum, are we saying we assume they are running on the TWN? Do you mean choose a cluster and shard?